shared/menubar/index.render.js.flow

```diff
@@ -1,35 +1,21 @@
 /* @flow */
 import React, {Component} from 'react'
 
-export type RenderProps = {
+import type {Props as FolderProps} from '../folders/render'
+
+export type Props = {
```

Comment:
RE: our conversation on explicit types, let's add `$Shape<...>` here.

You'll get some type errors that should be fixed too.